tests/consensus_tests/auth_tests/test_jwt_validation.py

```diff
@@ -833,23 +935,7 @@ def req():
 
 
 def test_rename_alias():
-    alias_names = [random_str() for _ in range(10)]
-
-    for alias in alias_names:
-        requests.post(
-            f"{REST_URI}/collections/aliases",
-            json={
-                "actions": [
-                    {
-                        "create_alias": {
-                            "collection_name": COLL_NAME,
-                            "alias_name": alias,
-                        }
-                    }
-                ]
-            },
-            headers=API_KEY_HEADERS,
-        ).raise_for_status()
+    alias_names = [random_str() for _ in range(12)]
```

Comment:
Suggestion: maybe it worth to use an endless iterator instead of hardcoding a number in every test:

```python
alias_names = iter(random_str, None)
```

Or create a global constant:
```python
MAX_CALLS_IN_CHECK_ACCESS = 12
…
def test_rename_alias():
    alias_names = [random_str() for _ in range(MAX_CALLS_IN_CHECK_ACCESS)]
```